### PR TITLE
Added rails instructions

### PR DESCRIPTION
**Pull request by finack**
And added a bit more in the overview section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,20 +92,48 @@ Add the following rack middleware to your app:
...
 with `$ACME_KEY` as the content.
 
-Please add any other language/framework by openin a Pull Request.
+Please add any other language/framework by opening a Pull Request.
+
+## Creating and deploy the sabayon app
 
-### Deploy the sabayon app
+In addition to configuring your application, you will also need to create
+a new Herkoku application which will run sabayon to create and update
```
Comment (dmathieu) [inline README.md]: `Herkoku` -> `Heroku`

File: README.md [hunk 2]
```diff
@@ -79,20 +92,48 @@ Add the following rack middleware to your app:
       end
     end
 
-### Other
+### Rails Applications
```
Comment (dmathieu) [inline README.md]: This is based on the ruby example (the ruby one could work with rails apps actually). How about putting it before ruby, so people don't start including ruby and then only notice there is something doable specifically for rails.

**Comment by finack** [comment]
Hi @dmathieu. Updated based on your feedback. Let me know if if you have other comments!

**Comment by dmathieu** [comment]
Thank you Peter.

**Pull request merged**